_sips/sips/2017-07-12-right-associative-by-name-operators.md

```diff
@@ -3,7 +3,7 @@ layout: sip
 discourse: true
 title: SIP-NN - Right-Associative By-Name Operators
```

Comment:
Are you sure this is SIP-34? I think it's 33.

Reply:
the 33 is match infix?

http://docs.scala-lang.org/sips/make-types-behave-like-expressions.html

Reply:
and thank you for the git compliment! I will frame it ((((o:::
I had THE best teacher, with incredible patience and presence = @heathermiller <3

Reply:
Yup, but since the numbers cannot be sequential because there were other SIPs numbered in the meanwhile. This would be SIP 36, if I'm not mistaken!

Reply:
![screen shot 2017-10-27 at 13 53 28](https://user-images.githubusercontent.com/31061184/32102839-502c228a-bb1e-11e7-86f8-5434fb76e76c.png)
I don't know if you are wrong, but contradictory for sure (o:

Reply:
Oh my! I read the title of the SIP wrong. Yeah, this is correct!